pkg/apis/kubermatic/v1/settings.go

```diff
@@ -68,6 +68,9 @@ type SettingSpec struct {
 
 	EnableOIDCKubeconfig bool `json:"enableOIDCKubeconfig"` //nolint:tagliatelle
 
+	// EnableClusterBackup enables the Cluster Backup feature in the dashboard.
```

Comment:
Since this is an optional field, would strongly recommend to turn into a pointer. Also pointers help a lot in case of migration and to detect whether a particular feature was really disabled by purpose or by default